### PR TITLE
add header and hamburger menu

### PR DESCRIPTION
**Pull request by cezaryskura**
- add header
- add hamburger menu
- remove mixin for buttons, add big red button instead

### CODE REVIEW

File: src/views/Header.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <header>
+    <Nav />
+    <router-link to="/">
+      <logoT class="my-4" />
+    </router-link>
+    <RouterView :key="$route.fullPath" />
```
Comment (IAfanasovMob) [inline src/views/Header.vue:7]: I feel like ` <RouterView :key="$route.fullPath" />` is not a part of a header

File: src/components/Nav.vue [hunk 2]
```diff
@@ -0,0 +1,176 @@
+<template>
+  <nav class="header__nav w-100 position-absolute">
+    <ul class="header__navbar px-0">
+      <li class="header__icon">
+        <a href="#" class="header__icon">
```
Comment (IAfanasovMob) [inline src/components/Nav.vue:5]: I was sure screen readers are confused with `a` tag without meaningless `href` value. semantically it is also a bit confusing.

File: src/components/Nav.vue [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+                    font-scale="2"
+                    class="dropdown__menu-link-icon"
+                  ></b-icon-box-arrow-right>
+                  <span class="dropdown__menu-link-text">Log in</span>
+                </router-link>
+              </li>
+
+              <li class="dropdown__menu-item py-3 ml-5">
+                <div class="dropdown__menu-link-text--small text-left">
+                  <router-link to="/algemene-voorwaarden"
```
Comment (IAfanasovMob) [inline src/components/Nav.vue:72]: `/algemene-voorwaarden` -> `routes` constant

File: src/components/Nav.vue [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+                    >Terms and conditions</router-link
+                  >
+                </div>
+              </li>
+              <li class="dropdown__menu-item py-3 ml-5">
+                <div class="dropdown__menu-link-text--small text-left">
+                  <router-link to="/algemene-voorwaarden"
```
Comment (IAfanasovMob) [inline src/components/Nav.vue:79]: `routes` constant

File: src/components/Nav.vue [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+}
+
+button {
+  background-color: transparent;
+  border: 0;
+}
+
+ul {
+  list-style-type: none;
+}
```
Comment (IAfanasovMob) [inline src/components/Nav.vue:118]: those styles for the tags would be applied to all the app. could lead to a confusion - app wide styles in a place for navigation component

File: src/components/Nav.vue [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+  right: 0;
+  z-index: 3;
+}
+.dropdown__menu-item > a {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  color: red !important;
+}
+.dropdown__menu-link-text {
```
Comment (IAfanasovMob) [inline src/components/Nav.vue:131]: duplication 😱 
```
.dropdown {
   &__menu-item > a {
   }
   &__menu-link-text {
   }
}
```

File: src/views/Header.vue (hunk 1 above)
Comment (cezaryskura) [reply src/views/Header.vue:7]: done

File: src/components/Nav.vue (hunk 3 above)
Comment (cezaryskura) [reply src/components/Nav.vue:72]: done

File: src/components/Nav.vue (hunk 5 above)
Comment (cezaryskura) [reply src/components/Nav.vue:118]: good one

File: src/components/Nav.vue (hunk 6 above)
Comment (cezaryskura) [reply src/components/Nav.vue:131]: Yes I know...it was done intentionally.. I wasn't able to commit. I was getting error:

.dropdown__menu-link-text {
  &--small > a:visited {

should be before 
.dropdown__menu-item > a {
and after 
.dropdown__menu-item > a:visited {

File: src/views/Home.vue [hunk 7]
```diff
@@ -58,11 +58,11 @@ export default {
...
+//   padding-bottom: 6em;
 
-  > a {
-    padding: 0.75em 1em !important;
-  }
-}
+//   > a {
+//     padding: 0.75em 1em !important;
+//   }
+// }
```
Comment (IAfanasovMob) [inline src/views/Home.vue:67]: delete?

**Pull request merged**